src/stylesheets/elements/_labels.scss

```diff
@@ -0,0 +1,13 @@
+.usa-label{
+  background-color: $color-gray;
+  font-size: rem(14px);
+}
+
+.usa-label-small {
+  @extend .usa-label;
+  font-size: rem(11px);
+}
+
```

Comment:
@carlosvalle - Thanks for the update! I am wondering if we can user test this size. Curious about readability at this size for people with low vision or glasses.